Modules/WorkingTree/WorkingTreeModel.cpp

```diff
@@ -23,26 +23,6 @@
 #include "WorkingTreeDirItem.h"
 #include "WorkingTreeFileItem.h"
 
-#ifdef Q_OS_WIN
-
-#include <windows.h>
-
-QIcon getWindowsIcon( const QString& pathName )
-{
-#if QT_VERSION < QT_VERSION_CHECK(5,0,0)
```

Comment:
Please don't drop this code. Rather port it using QtWinExtras (alternatives should be available nowadays)...